notebooks/ch-states/introduction.ipynb

```diff
@@ -38,39 +38,13 @@
     "To get you started on your journey towards quantum computing, let's test what you already know.\n",
     "\n",
     "\n",
-    "<!-- ::: q-block.exercise -->\n",
-    "\n",
     "### Quick Exercise\n",
     "\n",
-    "<!-- ::: q-quiz(goal=\"bitdescription\") -->\n",
-    "\n",
-    "<!-- ::: .question -->\n",
-    "\n",
     "Which of the following is the correct description of a *bit*?\n",
     "\n",
-    "<!-- ::: -->\n",
-    "\n",
-    "<!-- ::: .option(correct) -->\n",
-    "\n",
-    "The smallest unit of information: either a ```0``` or a ```1```.\n",
-    "\n",
-    "<!-- ::: -->\n",
-    "\n",
-    "<!-- ::: .option -->\n",
-    "\n",
-    "A blade used by a carpenter.\n",
-    "\n",
-    "<!-- ::: -->\n",
-    "\n",
-    "<!-- ::: .option -->\n",
-    "\n",
-    "Something you put in a horse's mouth.\n",
-    "\n",
-    "<!-- ::: -->\n",
-    "\n",
-    "<!-- ::: -->\n",
-    "\n",
-    "<!-- ::: -->\n"
+    "- The smallest unit of information: either a ```0``` or a ```1```.\n",
```

Comment:
@frankharkins - I'm curious if the intention here is to remove the interactive quiz. I'm seeing this in the preview branch:

<img width="542" alt="Screen Shot 2021-12-20 at 10 48 10 AM" src="https://user-images.githubusercontent.com/6276074/146817673-91b48ac1-e617-4ba2-8eb8-efd606128576.png">

Are you seeing the same?

https://platypus-pr-237.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/ch-states/introduction

Reply:
Yeah that's right; technically all are correct, but our quiz only supports one correct answer so I decided to remove it here. This page is really superseded by the intro course and we might want to think about removing it from platypus at some point in the future.